src/services/langflow_file_service.py

```
@@ -117,6 +117,14 @@ async def delete_user_file(self, file_id: str) -> None:
             )
         resp.raise_for_status()
 
+    async def delete_all_files(self, file_id: str) -> None:
+        """Delete a file by id using v2: DELETE /api/v2/files/{id}."""
+        import logging
+        # NOTE: use v2 root, not /api/v1
+        logger.debug("[LF] Delete (v2) -> /api/v2/files/{id}", file_id=file_id)
+        resp = await clients.langflow_request("DELETE", f"/api/v2/files")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)